src/layers/PolygonSeries.js

```diff
@@ -200,11 +204,15 @@ export class PolygonSeries extends PaletteMixin(CoverageMixin(L.Layer)) {
    */
   set time (val) {
     let old = this.time
-    this._axesSubset.t.coordPref = val ? val.toISOString() : undefined
+    this._axesSubset.t.coordPref = val ? new Date(asTime(val)).toISOString() : undefined
     
     this._loadCoverageSubset()
     if (old === this.time) return
-    this.redraw()
+    for (let layer of this._geojson.getLayers()) {

```

Comment:
> I've modified the set time methods of the (Multi)PolygonSeries to change the fillColor of the GeoJSON directly - I was finding that polygons were disappearing with a full redraw(), and this more direct method seems to work fine.

That's a good change. Why not move this into `_updatePolygon()` (and move the method up below `_addPolygon`) and call that method in set time? `redraw()` could then directly call what was in the old `_updatePolygon()` implementation.